### PR TITLE
Postcfp

### PR DESCRIPTION
**Pull request by MBcode**
might as well get the tail end of the last changes in, w/it drops the compat-tool subheading if none available

### CODE REVIEW

File: alpha/website/js/dataset2/obj-exchange.js [hunk 1]
```diff
@@ -403,7 +403,7 @@ this.raw_json = j;
 
         <div class="col-8">
             <div class="row">
-                <span class="font-weight-bold font-heavy"><br><h6>  ${html_name} </h6></span>
+                <span class="font-weight-bold font-heavy"><br><h6>  ${html_name} </h6><br></span>
```
Comment (valentinedwv) [inline alpha/website/js/dataset2/obj-exchange.js]: try col-12

**Review by valentinedwv — CHANGES_REQUESTED** [review]
PR#46 addresses the tools, too.
and try col-12, or a padding or margin bottom instead of a br.
mb-4 or pb-4

File: alpha/website/js/dataset2/obj-exchange.js (hunk 1 above)
Comment (MBcode) [reply alpha/website/js/dataset2/obj-exchange.js]: after js cache updating mb-2 w/top as well: mt-2 did not give the same result; not sure how col helps w/vertical spacing

Comment (MBcode) [reply alpha/website/js/dataset2/obj-exchange.js]: after js cache, now seeing some change w/mb-2 and mt-2, not same, but assume ok?

**Pull request merged**